layouts/shortcodes/embed-pdf.html

```diff
@@ -0,0 +1,111 @@
+<style>
+  .pdfContainer {
+    width: 100%;
+    height: auto;
+
+  }
+
+  #the-canvas {
+    border: 1px solid black;
+  }
+
+  .pdfNav {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 0 !important;
+    align-items: center;
+  }
+
+  .pdfBtn {
+    border-radius: 50%;
+    width: 40px;
+    height: 40px;
+    border: none;
+    margin: 2px;
+  }
+</style>
+
+<div class="pdfContainer">
+  <hr>
+  <div class="pdfNav">
+    <div class="paginations">
+      <button class="pdfBtn" id="prev">
+        <b>&lt</b>
+        </button>
+          <b id="currentPageNo"></b>/<b id="totalPages"></b>
+          <button class="pdfBtn" id="next">
+            <b> &gt</b>
+           </button>
+    </div>
+
+    <a href="{{.Site.BaseURL}}pdfs/{{ .Get "pdfName" }}" target="_blank">View PDF</a>
+  </div>
+  <canvas id="the-canvas"></canvas>
+</div>
+<script>
+  var pdf = "{{.Site.BaseURL}}" + "pdfs/" + "{{ .Get "pdfName" }}";
+  const prevBtn = document.getElementById("prev");
+  const nextBtn = document.getElementById("next");
+  const currentPageNoElement = document.getElementById("currentPageNo");
+  const loadingTask = pdfjsLib.getDocument(pdf);
+
+  loadingTask.promise.then((pdf) => {
+    var currentPage = 1;
+
+    //set toal pages
+    const totalPages = pdf._pdfInfo.numPages;
+    const totalPagesElement = document.getElementById("totalPages");
+    totalPagesElement.innerHTML = totalPages;
+    console.log(totalPages)
+    // function to set current page number
+    setCurrentPageNo();
+    function setCurrentPageNo() {
+      currentPageNoElement.innerHTML = currentPage;
+    }
+    //on clicking prev btn
+    prevBtn.addEventListener("click", () => {
+      if (currentPage > 1) {
+        currentPage--;
+        setCurrentPageNo();
+        renderPage(currentPage);
+
+      }
+    });
+    //on clicking next btn
+    nextBtn.addEventListener("click", () => {
+      if (currentPage < totalPages) {
+        currentPage++;
+        setCurrentPageNo();
+        renderPage(currentPage);
+
+      }
+    });
+    renderPage(currentPage);
+
+    //function to render given page number
+    function renderPage(no) {
+      pdf.getPage(no).then((page) => {
+        console.log(page);
+        var scale = 1.5;
+        var viewport = page.getViewport({ scale: scale });
+        var outputScale = window.devicePixelRatio || 1;
+        var canvas = document.getElementById("the-canvas");
+        var context = canvas.getContext("2d");
+        canvas.width = viewport.width;
+        canvas.height = viewport.height;
+        canvas.style.width = "100%";
+        canvas.style.height = "100%";
```

Comment:
Something seems amiss here. The PDF appears cropped and there is no way to scroll inside the PDF container. 🤔 I don't have the time to dig into this now. Think you can explore what could be wrong here?

![Screenshot of the embedded PDF on a page. The PDF is zoomed in and some text appears to flow off-screen. There are no ways to scroll or move the PDF around.](https://user-images.githubusercontent.com/4721034/157507233-bb8abe65-7aad-4ec5-b742-ab4682cd1782.png "Screenshot of the embedded PDF on a page. The PDF is zoomed in and some text appears to flow off-screen. There are no ways to scroll or move the PDF around.")

Reply:
I don't know why this happened. It worked fine for me 🤔
I will dig into it and will fix it.

Reply:
@Neha9849 Curious, did you try it on Firefox or Chromium/Chrome?

Reply:
I checked on chrome

Reply:
I was using Firefox. I am away from my keyboard for the evening, but I can try again from Chromium tomorrow.

Reply:
> Something seems amiss here. The PDF appears cropped and there is no way to scroll inside the PDF container. 🤔 I don't have the time to dig into this now. Think you can explore what could be wrong here?

I have made some changes to fix this one. I think it will work fine now, I could not test it as i am not using Firefox. Could you check it on Firefox once.
Also, i have made the changes as requested.

Reply:
Fixed in commit d3dc9be2086c3a30dd966fbe84bfe046d36fbc6c.